string ALTITUDE
string AZIMUTH
---
TelescopeInfo status
